### ISSUE TITLE
Request For Feedback

### ISSUE THREAD

**Issue by r3wt** [gabordemooij/redbean#443]
@gabordemooij 

As you probably know by now, i'm a religious zealot when it comes to redbean. I absolutely love the library. At work, i built a small library called "Filter" for processing huge medical forms and generating redbean models. We loved it so much it replaced all form processing and validation in our application.

 I have received permission to release an open source version of it, so i've begun doing so.

In a nutshell it just automates the entire process of filtering and validating input and building the redbean model. I've just pushed the first commit. Its basically just the library and a small example for now. Just to get the idea out there and see if people think its useful.

https://github.com/r3wt/RedBeanFVM

Feedback + Contribution welcomed (and appreciated). Cheers!

**Comment by gabordemooij** [comment]
Hi there,

Looks wonderful, I can imagine how such a tool boosts productivity.
Thanks for sharing this with us.
If you provide me with a nice description ala..


RedBean FVM
===========

Does this... etc does that etc...


I will put it on the RedBeanPHP website :)

cheers,
Gabor

**Comment by r3wt** [comment]
Thanks for your reply Gabor. I will notify you once i have a stable release candidate ready. I actually just threw it all together this morning to get the idea out there. Thanks for your time!

**Comment by r3wt** [comment]
@gabordemooij Ok Gabor. I've finished up the library. Now i'm just doing some testing just to make sure i didn't miss any edge cases or make any mistakes. I attempted to write a succinct explanation of the plugin, but i'm not sure i did a good job of explaining it:

RedBeanFVM
===========

RedbeanFVM makes Filtering, Validating , and Generating RedBean Models easy, by taking your blank Model, some rules, and the data source and generating a fully constructed RedBean Model.

You can see a list of examples in the * [README](https://github.com/r3wt/RedBeanFVM/blob/master/README.md)

**Comment by gabordemooij** [comment]
added http://www.redbeanphp.com/plugins

**Comment by r3wt** [comment]
Alright cool. i'm going to go ahead and close this now. Thanks! Long live RedBeanPHP!
